distribution/docker/solrcloud/docker-compose.yml

```diff
@@ -5,22 +5,23 @@ version: '3.7'
 services:
 
   zoo:
-    image: zookeeper:3.4
+    image: zookeeper:3.5
     container_name: zoo
     restart: always
     ports:
       - "2181:2181"
+      - "2182:8080"
```

Comment:
Why not `2182:2182`?

Reply:
`8080` is the port Zookeeper opens but that is a common developer default port.  I am exposing it on 2182 instead to keep it close to the other exposed Zookeeper port.